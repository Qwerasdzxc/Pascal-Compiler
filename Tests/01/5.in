494.382 416.171

