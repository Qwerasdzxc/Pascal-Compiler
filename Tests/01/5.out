910.55 78.21 1.19